certificate-analytics-service/app/src/main/java/se/inera/intyg/certificateanalyticsservice/infrastructure/configuration/MessagingConfig.java

```diff
@@ -0,0 +1,53 @@
+package se.inera.intyg.certificateanalyticsservice.infrastructure.configuration;
+
+import jakarta.jms.ConnectionFactory;
+import jakarta.jms.Queue;
+import java.util.List;
+import org.apache.activemq.ActiveMQConnectionFactory;
+import org.apache.activemq.command.ActiveMQQueue;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jms.annotation.EnableJms;
+import org.springframework.jms.connection.CachingConnectionFactory;
+import org.springframework.jms.core.JmsTemplate;
+
+@Configuration
+@EnableJms
+public class MessagingConfig {
+
+  @Value("${certificate.event.queue.name}")
+  private String eventQueueName;
+
+  @Value("${spring.activemq.broker-url}")
+  private String brokerUrl;
+
+  @Value("${spring.activemq.user}")
+  private String amqUser;
+
+  @Value("${spring.activemq.password}")
+  private String amqPassword;
+
+  @Bean
+  public Queue eventQueue() {
+    return new ActiveMQQueue(eventQueueName);
+  }
+
+  @Bean
+  public ConnectionFactory cachingConnectionFactory() {
```

Comment:
Is it necessary to configure this ourself? Can´t the autoconfigure support in Springbok solve this for us. I'm thinking of us creating the ActiveMQConnectionFactory and having to use spring.activemq.* properties.